src/System.Buffers/ref/Configurations.props

```diff
@@ -1,15 +1,8 @@
 ﻿<?xml version="1.0" encoding="utf-8"?>
 <Project ToolsVersion="14.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
   <PropertyGroup>
-    <PackageConfigurations>
-      netstandard;
-      netstandard1.1;
-      net45;
-    </PackageConfigurations>
     <BuildConfigurations>
-      $(PackageConfigurations);
-      uap;
-      netfx;
+      netstandard;
```

Comment:
Should we make this netcoreapp and uap? Once this becomes part of netstandard vnext we will be forced to do that.

Reply:
I will try to make this netcoreapp and uap. There was a reference to this ref assembly that was preventing me from doing so, but I will figure it out.

Reply:
The reason why I made this netstandard was because System.IO.Pipelines references System.Buffers and now with this change IO.Pipelines ref is built for netstandard, should we wait until this becomes part of vnext to make this ref netcoreapp and uap? or should we just build Pipelines ref assembly for netcoreapp and uap?

Reply:
Lets keep it simple for now and stick with netstandard.